packages/backend/src/rates/service.ts

```diff
@@ -54,7 +54,7 @@ export function createRatesService(deps: ServiceDependencies): RatesService {
 
 class RatesServiceImpl implements RatesService {
   private axios: AxiosInstance
-  private cachedRates: CacheDataStore
+  private cachedRates: CacheDataStore<string>
```

Comment:
ah, we should have just stored the actual object with the generic type from the start (instead of parsing in and out) but we can keep this for now